src/App/Pages/Vault/GroupingsPage/GroupingsPage.xaml.cs

```diff
@@ -92,12 +92,12 @@ protected async override void OnAppearing()
                 else if (message.Command == "syncCompleted")
                 {
                     await Task.Delay(500);
-                    Device.BeginInvokeOnMainThread(() =>
+                    Device.BeginInvokeOnMainThread(async() =>
                     {
                         IsBusy = false;
                         if (_vm.LoadedOnce)
                         {
-                            var task = _vm.LoadAsync();
+                            await _vm.LoadAsync();
```

Comment:
Did this need to be changed for a reason?

Reply:
This was to look about await inside BeginInvokeOnMainThread but since it makes no difference i pushed now a commit to apply changes.